databases/psql_restore_slave.rst

```diff
@@ -0,0 +1,97 @@
+Slave Recovery
+==============
+
+What to do if Replication Streaming on Postgres fails
+-----------------------------------------------------
+
+Important! Read this section first before doing anything!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this section, very powerful commands are described. So before execute anything, be 100% sure that you are on the right server!
+Else you could cause a lot of damage. 
+
+Master --> db01c
+
+Slave --> db01a
+
+
+Check The Delation
+^^^^^^^^^^^^^^^^^^
+
+First of all you shuld check the delation. If the delation is critical nagios or Backoffice will inform you about it.
+This will look like this:
+
+.. code::
+
+   POSTGRES_HOT_STANDBY_DELAY CRITICAL: DB postgres (host:db01slave) 147866242312 seconds
+
+if the number (delation in seconds) is extremly high then you know something must be wrong with the Replication Streaming
```

Comment:
*Streaming Replication* → *replication* (We use a combination of WAL replication and streaming replication but since it doesn't matter in this context I'd not mention it.)

Reply:
done